types/react-table/index.d.ts

```diff
@@ -601,9 +601,12 @@ export interface RowRenderProps extends Partial<RowInfo> {
 }
 
 export interface RowInfo {
-    /** Original row from your data */
+    /** Materialized row of data */
     row: any;
```

Comment:
I believe the documenation states this is of type object.

Reply:
This PR is not addressing the underlying type.

Reply:
Sounds good to me, you can ignore this comment.